src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.ManufacturerDao;
+import mate.jdbc.lib.ManufacturerDaoImpl;
+import mate.jdbc.models.Manufacturer;
+
 public class Main {
     public static void main(String[] args) {
-
+        ManufacturerDao manufacturerDao = new ManufacturerDaoImpl();
```

Comment:
```suggestion
        ManufacturerDao manufacturerDao =
                (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
```
We have to use injector